src/vs/platform/contextkey/common/contextkey.ts

```diff
@@ -25,6 +25,9 @@ CONSTANT_VALUES.set('isEdge', isEdge);
 CONSTANT_VALUES.set('isFirefox', isFirefox);
 CONSTANT_VALUES.set('isChrome', isChrome);
 CONSTANT_VALUES.set('isSafari', isSafari);
+// --- START POSITRON ---
```

Comment:
Nit, we've used `Start Positron` and `End Positron` mixed case for these regional comment markers.

Reply:
I'm not sure why I thought we always used capitals. 🤯